variables.md

```diff
@@ -1,204 +1,195 @@
-
+By [codecademy.com](https://codecademy.com)
 
 
 ## Mathematical Assignment Operators
 
 Variables and math operators can be used to calculate new values and assign them to a variable.
 
-```
-let w = 4;
-w = w + 1;
+```js
+let num = 7;
+num = num + 2;
 
-console.log(w); // Output: 5
+console.log(w); // Output: 9
 ```
+from the example above the variable `num` was assigned the value of `7`. Then the `num` variable was reassigned the addition of itself and `1`. 
 
-In the example above, a variable ```w``` is created with the number ```4``` assigned to it. The following line, ```w = w + 1```, increases the value of ```w``` from ```4``` to ```5```.
-
-Another way that ```w``` could have been reassigned after performing some mathematical operation on it is to use built-in mathematical assignment operators. We could re-write the code above to be:
+Use built-in mathematical assignment operators is another way that the `num` variable can be reassigned. We could re-write the code above to be:
 
-```
-let w = 4;
-w += 1;
+```js
+let num = 7;
+num += 2;
 
-console.log(w); // Output: 5
+console.log(num); // Output: 9
 ```
 
-In the second example, the ```+=``` assignment operator is used to reassign ```w```. We’re performing the mathematical operation of the first operator ```+``` using the number to the right, then reassigning ```w``` to the computed value.
+In the second example, the `+=` assignment operator is used to reassign `num`. We’re performing the mathematical operation of the first operator `+` using the number to the right, then reassigning `num` to the computed value.
 
-We also have access to other mathematical assignment operators: ```-=```, ```*=```, and ```/=``` which work in a similar fashion.
+Other mathematical assignment operators we also have access to are: `-=`, `*=`, and `/=` which work in a similar fashion.
 
-```
-let x = 20;
-x -= 5; // Can be written as x = x - 5
-console.log(x); // Output: 15
+```js
+let num = 25;
+num -= 5; // Can be written as num = num - 5
+console.log(num); // Output: 20
 
-let y = 50;
-y *= 2; // Can be written as y = y * 2
-console.log(y); // Output: 100
+let numTwo = 50;
+numTwo *= 3; // Can be written as numTwo = numTwo * 3
+console.log(numTwo); // Output: 150
 
-let z = 8;
-z /= 2; // Can be written as z = z / 2
-console.log(z); // Output: 4
+let numThree = 8;
+numThree /= 2; // Can be written as numThree = numThree / 2
+console.log(numThree); // Output: 4
 ```
 
 ## The Increment and Decrement Operator
 
 The increment operator will increase the value of the variable by 1. The decrement operator will decrease the value of the variable by 1. For example:
 
-```
-let a = 10;
-a++;
-console.log(a); // Output: 11
-```
+The increment operator is used to increase the value of a variable by 1 while the decrement operator is used to decrease the value of a variable by 1.
 
+```js
+let num = 17;
+num++;
+console.log(num); // Output: 18
```

Comment:
It is important to note that if `num == 17`, `console.log(num++)` will log `17` and not `18`.